CHANGELOG.md

```diff
@@ -2,19 +2,30 @@
 
 ## Next release
 
+### Breaking changes
+
+This PR contains the [PDV refactor (#4111)](https://github.com/hasura/graphql-engine/pull/4111), a significant rewrite of the internals of the server, which did include some breaking changes:
+
+- The semantics of `where` have changed, according to the discussion in [issue 704](https://github.com/hasura/graphql-engine/issues/704#issuecomment-635571407). Namely: an explicit `null` value in a `where` expression will be treated as an error rather than resulting in the expression being evaluated to `True`. Meaning that `delete_users(where: {id: {_eq: $userId}}) { name }` will yield an error if `$userId` is `null` instead of deleting all users.
+- The validation of required headers has been fixed (closing #14 and #3659):
```

Comment:
Why is `The validation of required headers has been fixed` considered a breaking change?

Reply:
Yeah, you're right, this should rather be among the fixes. I can fix this in an hour or so.

Reply:
Because some queries might fail where they used to "work" before.
Imagine you do the following:
```
query {
  author {
    name
    articles {
      name
    }
  }
}
```
Now imagine that header `X-ARTICLE-VIEW-PERMIT` is required for the select permissions on table `articles`. That header **was not checked** before this patch, because we only checked the headers for the root table, which means **this query would have succeeded without the required header**. It would have returned no article, since the header was missing, and its value would have been evaluated to an empty string, but still: the query would have gone through. Now it will properly fail: a header is required to select from table articles, and you don't have it.